components/ReplacementsTable.tsx

```diff
@@ -65,10 +65,16 @@ const ReplacementsTable = ({ replacements }: Props) => {
     }));
   };
 
+  const dateParts = replacements.PLDate.split('.').map((datePart) =>
+    parseInt(datePart, 10)
+  );
```

Comment:
IMO since we know that it's just a date (predictable numbers in a decimal format), `parseInt` is too redundant
```suggestion
  const dateParts = replacements.PLDate.split('.').map((datePart) =>
    Number(datePart)
  );
```